### PR TITLE
Add support for config object

### PR DESCRIPTION
**Pull request by dcalhoun**
Allow for defining a config object for a plugin that contains a `hotkey` attribute.
```
{
  pluginName: {
    hotkey: 'Ctrl+;',
    other: true
  }
}
```

This allows a Hyper plugin to use this library, but fit it into an existing config object. I didn't test these changes in Hyper, but I did add test for the block of source that I altered. I plan to use this ability to expand `hyperterm-summon` with additional config options (e.g. hiding the app icon from the app switcher). Hope thats alright. 😄

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,8 +1,5 @@
 const { dialog, globalShortcut } = require('electron');
-
-function getShortcutFromConfig (key, config) {
-  return (config.hotkeys && config.hotkeys[key]) || config[`${key}Shortcut`];
-}
+import getShortcutFromConfig from './lib/index'
```
Comment (soutar) [inline index.js]: Could we change this to a `require` to keep it consistent with the others (the line above and the test you added)?

Comment (dcalhoun) [reply index.js]: 👍 Will do.

**Comment by soutar** [comment]
This is a good idea and I totally approve. Once the import is changed I'm happy to merge 👍

**Comment by soutar** [comment]
@dcalhoun I've added you as a collaborator. Could you push this on a branch of this repo? I want to add TravisCI for running the tests you added automatically on every PR. Being able to raise a PR against your branch would make that much easier 😄

**Comment by dcalhoun** [comment]
Closed in favor of #4.

**Pull request closed without merging**